<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DependenciesEditWidget</class>
 <widget class="QWidget" name="DependenciesEditWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dependencies</string>
  </property>
  <layout class="QVBoxLayout" name="loDependencies">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_10">
     <item>
      <widget class="QLabel" name="lblDependecies">
       <property name="text">
        <string>Dependencies of this test</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnDependenciesHideRight">
       <property name="maximumSize">
        <size>
         <width>13</width>
         <height>13</height>
        </size>
       </property>
       <property name="text">
        <string notr="true"/>
       </property>
       <property name="icon">
        <iconset resource="../../qHostMonitor.qrc">
         <normaloff>:/img/minus.gif</normaloff>
         <normalon>:/img/plus.gif</normalon>:/img/minus.gif</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>13</width>
         <height>13</height>
        </size>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QFrame" name="frmDependentTests">
     <property name="frameShape">
      <enum>QFrame::Panel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <property name="lineWidth">
      <number>2</number>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_11">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QTreeWidget" name="treeDependentTests">
        <property name="rootIsDecorated">
         <bool>false</bool>
        </property>
        <property name="columnCount">
         <number>2</number>
        </property>
        <attribute name="headerDefaultSectionSize">
         <number>150</number>
        </attribute>
        <column>
         <property name="text">
          <string>Dependent test</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Condition</string>
         </property>
        </column>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <property name="spacing">
         <number>0</number>
        </property>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QToolButton" name="btnDependTestAdd">
          <property name="maximumSize">
           <size>
            <width>22</width>
            <height>22</height>
           </size>
          </property>
          <property name="toolTip">
           <string>Show tests tree</string>
          </property>
          <property name="statusTip">
           <string>Show tests tree</string>
          </property>
          <property name="text">
           <string>...</string>
          </property>
          <property name="icon">
           <iconset resource="../../qHostMonitor.qrc">
            <normaloff>:/img/new.png</normaloff>:/img/new.png</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="btnDependTestDelete">
          <property name="maximumSize">
           <size>
            <width>22</width>
            <height>22</height>
           </size>
          </property>
          <property name="toolTip">
           <string>Show tests tree</string>
          </property>
          <property name="statusTip">
           <string>Show tests tree</string>
          </property>
          <property name="text">
           <string>...</string>
          </property>
          <property name="icon">
           <iconset resource="../../qHostMonitor.qrc">
            <normaloff>:/img/trash.png</normaloff>:/img/trash.png</iconset>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../qHostMonitor.qrc"/>
 </resources>
 <connections/>
</ui>
